apbs/chemistry/atom.py

```diff
@@ -20,39 +20,27 @@ def __init__(self, *args, **kwargs):
         """
         Arguments:
 
-        :param id: A unique identifier for this Atom
-        :type id: int
-        :param field_name: Specifies the type of PQR entry and should either
+        :param int id: A unique identifier for this Atom
```

Comment:
What versions of Sphinx support this combined type syntax?  Are they compatible with readthedocs?

Reply:
Version 1.7, https://www.sphinx-doc.org/en/1.7/domains.html?highlight=send_message, search the page for `send_message`.

Reply:
I can't find anywhere that states what version of Sphinx is supported in RTD.